packages/sx05re/emuelec/profile.d/99-emuelec.conf

```diff
@@ -22,11 +22,16 @@ SETTING="${1}"
 PLATFORM="${2}"
 ROM="${3}"
 
+EES=""
+
 #ROM
-ROM=$(echo [\"${ROM}\"] | sed -e 's|\[|\\\[|g' | sed -e 's|\]|\\\]|g' | sed -e 's|(|\\\(|g' | sed -e 's|)|\\\)|g')
-PAT="^${PLATFORM}${ROM}[.-]${SETTING}=(.*)"
+
+if [ ! -z "${ROM}" ]; then
```

Comment:
Technically this part is not needed. But it can't hurt to skip if the rom file is blank.